browser_history/generic.py

```diff
@@ -207,25 +237,33 @@ class Outputs:
     formats = ("csv", "json")
   
     # Use the below fields for all formatter implementations
-    fields = ("Timestamp", "URL")
+    history_fields = ("Timestamp", "URL")
+    bookmark_fields = ("Timestamp","URL","Title","Folder")
 
     def __init__(self):
-        self.entries = []
+        self.history_entries = []
+        self.bookmark_entries =[]
         # format map is used by the formatted method to call the right formatter
         self._format_map = {
             "csv": self.to_csv,
             "json": self.to_json,
             "jsonl": lambda: self.to_json(json_lines=True)
```

Comment:
You'll have to update this lambda function with the type parameter

Reply:
Oh right!